src/hoc/withEditing/withEditing.tsx

```diff
@@ -31,19 +35,23 @@ export const withEditing = (config: IWithEditing) => (Component: React.Component
     });
 
     @ReduxConnect(mapStateToProps, mapDispatchToProps)
-    class ComponentWithEditing extends React.Component<any, any> {
+    class ComponentWithEditing extends React.Component<IWithEditingProps> {
+        componentDidMount() {
+            callIfDefined(this.props.toggleDirtyComponent, config.isDirty);

```

Comment:
toggleDirtyComponent cant be undefined. you can remove callIfDefined

Reply:
removed